core/services/keeper/orm.go

```diff
@@ -91,7 +91,7 @@ RETURNING *
 	return errors.Wrap(err, "failed to upsert upkeep")
 }
 
-// Update the last keeper index for an upkeep
+// UpdateUpkeepLastKeeperIndex the last keeper index for an upkeep
```

Comment:
nit: UpdateUpkeepLastKeeperIndex "updates" the last keeper index for an upkeep

Reply:
oops!